thirdparty_builtin/CMakeLists.txt

```diff
@@ -87,7 +87,7 @@ if(ENABLE_TESTS)
     
     if(ENABLE_GTEST)
         # Enable builtin google test 
-        add_subdirectory(googletest-release-1.8.0)
+        add_subdirectory(googletest-release-1.8.0 ${PROJECT_BINARY_DIR}/blt/thirdparty_builtin/googletest-release-1.8.0)
```

Comment:
Does anyone know a more clever way of identifying the path that I want here other than hardcoding it?

Reply:
@robinson96 can you say a little more what scenario this is needed for? Are there two instances of BLT?